doc/fragments/@target_alphabet@_implicit_conversion_from_@source_alphabet@.hpp.in

```diff
@@ -0,0 +1,28 @@
+// generated from ${source_snippet}
+
+namespace seqan3::doxygen
+{
+/*!\details
+ *
+ * Normally, we don't allow implicit conversion of single argument constructors, but in this case we allow it,
+ * because seqan3::${target_alphabet} and seqan3::${source_alphabet} are interchangeable as they behave nearly the same (e.g. same ranks, same
+ * char to rank conversion).
```

Comment:
```suggestion
 * Normally, we do not allow implicit conversion of single argument constructors, but in this case we make an exception,
 * because seqan3::${target_alphabet} and seqan3::${source_alphabet} are interchangeable as they behave nearly the same (e.g. same ranks, same
 * char to rank conversion).
```

Reply:
done :)